docs/apis-tools/camunda-api-rest/specifications/sidebar.js

```diff
@@ -125,7 +157,7 @@ module.exports = [
   },
   {
     type: "category",
-    label: "Decision definition",
+    label: "Decision Definition",
```

Comment:
change-at-the-source, regression, me:

I think this is probably from #4178, and didn't get updated in even our own yaml file.

Reply:
https://github.com/camunda/camunda/pull/22037